<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ProcessViewerBase</class>
 <widget class="QWidget" name="ProcessViewerBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Process Viewer</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="mTitleLbl" >
     <property name="font" >
      <font>
       <family>Sans Serif</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="autoFillBackground" >
      <bool>true</bool>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth" >
      <number>3</number>
     </property>
     <property name="text" >
      <string>Process Viewer</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="mProcessTable" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="mRefreshBtn" >
       <property name="text" >
        <string>&amp;Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mTerminateBtn" >
       <property name="text" >
        <string>&amp;Terminate</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
